config/deploy/production.rb

```diff
@@ -1,7 +1,8 @@
-set :application, 'joegattnet_v3'
-set :branch,      'master'
-set :deploy_to,   '/home/deployer/apps/joegattnet_v3'
-set :rails_env,   'production'
+set :application,     'joegattnet_v3'
+set :branch,          'master'
+set :deploy_to,       '/home/deployer/apps/joegattnet_v3'
+set :rails_env,       'production'
+set :sidekiq_options, '/home/deployer/apps/joegattnet_v3/shared/config/sidekiq.yml'
```

Comment:
Line is too long. [83/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.